WeWatch/WeWatch/Splash/SplashView.swift

```diff
@@ -38,6 +38,7 @@ struct SplashView: View {
                 }
                 .font(.poppinsBold24px)
             }
+            .loader(isLoading: $viewModel.isLoading)
```

Comment:
We don’t need a loader on the splash view since the splash view itself serves as a loader.